### PR TITLE
chore: improve cli tools

### PR DESCRIPTION
**Pull request by mh-alahdadian**
- upgrade prettier and lint
- migrate to prettier-organize-imports and remove import-sort

### CODE REVIEW

**Review by mdeljavan — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  extends: ['react-app'],
+  ignorePatterns: ['dist', 'build', 'node_modules', '.storybook'],
```
Comment (m-davoodi) [inline .eslintrc.js:3]: Please also check for this path: 'storybook-static'

File: lint-staged.config.js [hunk 2]
```diff
@@ -1,14 +1,9 @@
...
+  '*.{ts,tsx,js,jsx}': (files) => {
     return [
-      `import-sort --write ${files.join(' ')}`,
-      `utsdx lint --max-warnings 0 ${files.join(
-        ' ',
-      )} --ignore-pattern '!.storybook'`,
+      `eslint --max-warnings 0 ${files.join(' ')}`,
       'lerna run type:check',
     ];
   },
```
Comment (m-davoodi) [inline lint-staged.config.js:7]: "*.{ts,tsx,js,jsx}":
        "yarn eslint --max-warnings 0" && lerna run type:check,

File: .eslintrc.js (hunk 1 above)
Comment (mh-alahdadian) [reply .eslintrc.js:3]: will be done in next PR

File: lint-staged.config.js (hunk 2 above)
Comment (mh-alahdadian) [reply lint-staged.config.js:7]: that wouldnt work

**Pull request merged**